governance/polls/MIP10c9-SP14 - Whitelist Yearn Finance on YFIUSD Oracle - November 30, 2020.md

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
So I changed the template slightly given recent confusion around polling methods. Updated version can be found here: https://github.com/makerdao/community/blob/master/governance/polls/templates/MIP10c9-SP%23%20-%20Whitelist%20%24whitelisted_entity%20on%20%24whitelisted_oracle%20Oracle%20-%20%20%24date_MONTH_DD%2C_YYYY.md